PicaSWB.js

```diff
@@ -39,6 +48,16 @@ var nachnameMapping = {
 };
 var nameMapping = {
```

Comment:
IxTheo spezifisches Verfahren bei der Autoren PPN-Verknüpfung, das vllt. in Master-Datei in modifizierter Form später übernommen werden könnte:
Falls Autorenmapping nicht greift, wird noch manuell in ognd recherchiert (siehe code unten), ob ein entsprechender Autorensatz doch vorhanden ist. Fall ja, wird die PPN mit copy&paste aus ognd übernommen. Um die erzeugten Links ohne copy&paste anklickbar zu machen,  haben wir während der Testphase in IxTheo folgenden Editor benutzt: http://www.w3schools.com/js/tryit.asp?filename=tryjs_events 
Ganz praktisch, da der Zotero-Export mit drag&drop im "Edit This Code"-Bereich funktioniert  und im "Result"-Bereich die ognd direkt aufgerufen werden kann, ohne dass man das Browsingfenster wechseln muss: 
Durch diese neue Methode in Kombination mit dem "nameMapping"  ist der Anteil von Autorenverknüpfungen bis jetzt von ca. 30 % auf über 60% angestiegen. Außerdem kann die Suchabfrage in ognd je nach Bedarf individuell angepasst werden, um nur eine bestimmte Teilmenge von Autorensätze der GND abzufragen. (Berufsbezeichnung, GND-Systematik, Geburtsdatum etc.)
` 
//Autorenmapping mit ognd kontrollieren. ogndppn1 = Nachname, Vorname; gondppn2 = Nachname ; 0900 Kommentarfeld http://swbtools.bsz-bw.de/cgi-bin/help.pl?cmd=kat&val=0999&regelwerk=RDA&verbund=SWB 

     if (item.creators) { Zotero.write( "0999 AUTOR: " + item.creators + creator.lastName + ", " + creator.firstName + " (TUIXTH-Test)" + "\n" + "\n" + "\n"); }
		
		 var ogndppn1 = 'http://swb.bsz-bw.de/DB=2.104/SET=1/TTL=1/CMD?SGE=&ACT=SRCHM&MATCFILTER=Y&MATCSET=Y&NOSCAN=Y&PARSE_MNEMONICS=N&PARSE_OPWORDS=N&PARSE_OLDSETS=N&IMPLAND=Y&NOABS=Y&ACT0=SRCHA&SHRTST=50&IKT0=1004&TRM0=' +  [creator.lastName + ", " + creator.firstName] + '&ACT1=*&IKT1=2057&TRM1=3.*&ACT2=*&IKT2=8991&TRM2=19**&ACT3=%2B&IKT3=4060&TRM3=tpv*&ACT4=%2B&IKT4=8991&TRM4=theol* neutestament*&ACT5=*&IKT5=1004&TRM5=' +  [creator.lastName + ", " + creator.firstName] ;
		 var ogndppn2 = 'http://swb.bsz-bw.de/DB=2.104/SET=1/TTL=1/CMD?SGE=&ACT=SRCHM&MATCFILTER=Y&MATCSET=Y&NOSCAN=Y&PARSE_MNEMONICS=N&PARSE_OPWORDS=N&PARSE_OLDSETS=N&IMPLAND=Y&NOABS=Y&ACT0=SRCHA&SHRTST=50&IKT0=1004&TRM0=' +  [creator.lastName ] + '&ACT1=*&IKT1=2057&TRM1=3.*&ACT2=*&IKT2=8991&TRM2=19**&ACT3=%2B&IKT3=4060&TRM3=tpv*&ACT4=%2B&IKT4=8991&TRM4=theol* neutestament*&ACT5=*&IKT5=1004&TRM5=' +  [creator.lastName] + "%2F2"  ;		 
		
		
		 if (item.creators) { Zotero.write("\n" + "MAPPING_BEDINGUNG > NACHNAME, VORNAME | TPV_sn3.* | IN OGND PPN PRÜFEN | LINK:   ".fontcolor("green") + ogndppn1.link(ogndppn1) + "MAPPING_BEDINGUNG > NACHNAME | TPV_rlsTHEOL | IN OGND PPN PRÜFEN | LINK:   ".fontcolor("green") + ogndppn2.link(ogndppn2) + "\n"+ "\n");}
		 `